lua/go/asyncmake.lua

```diff
@@ -369,6 +369,8 @@ M.runjob = function(cmd, runner, args, efm)
           }
         end
         vim.fn.setqflist({}, ' ', opts)
+      elseif vim.fn.getqflist({ title = 0 }).title == cmdstr then
+         vim.api.nvim_command([[:cclose]])
```

Comment:
I think you may also want to clear quickfix list by something like `vim.fn.setqflist({}, ' ', {lines = {}})`

Reply:
`vim.fn.setqflist({}, ' ', {lines = {}})` by itself won't close the qf windown